homeassistant/components/sleepiq/__init__.py

```diff
@@ -77,11 +77,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         raise ConfigEntryNotReady(str(err) or "Error reading from SleepIQ API") from err
 
     coordinator = SleepIQDataUpdateCoordinator(hass, gateway, email)
+    pause_coordinator = SleepIQPauseUpdateCoordinator(hass, gateway, email)
 
     # Call the SleepIQ API to refresh data
     await coordinator.async_config_entry_first_refresh()
+    await pause_coordinator.async_config_entry_first_refresh()
 
-    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = (coordinator, pause_coordinator)
```

Comment:
It would be cleaner to make a `dataclass` to store this. Example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/lookin/models.py#L13